templates/resources.md.tmpl

```diff
@@ -0,0 +1,77 @@
+    {{ else if eq .Name "mongodbatlas_privatelink_endpoint_service" }}
+        {{ tffile "examples/mongodbatlas_privatelink_endpoint/aws/cluster/main.tf" }}
+    {{ else if eq .Name "mongodbatlas_cloud_provider_access_authorization" }}
+        {{ tffile "examples/mongodbatlas_cloud_provider_access/aws/main.tf" }}
+    {{ else if eq .Name "mongodbatlas_cloud_provider_access_setup" }}
+        {{ tffile "examples/mongodbatlas_cloud_provider_access/aws/main.tf" }}
+    {{ else if eq .Name "mongodbatlas_federated_settings_org_config" }}
+        {{ tffile "examples/mongodbatlas_federated_settings_org_role_mapping/main.tf" }}
+    {{ else if eq .Name "mongodbatlas_cloud_backup_snapshot" }}
+        {{ tffile "examples/mongodbatlas_cloud_backup_snapshot_export_job/main.tf" }}
+    {{ else if eq .Name "mongodbatlas_api_key" }}
+        {{ tffile (printf "examples/%s/create-and-assign-pak/main.tf" .Name )}}
+    {{ else if eq .Name "mongodbatlas_backup_compliance_policy" }}
+    {{ else if eq .Name "mongodbatlas_event_trigger" }}
+    {{ else if eq .Name "mongodbatlas_access_list_api_key" }}
+    {{ else if eq .Name "mongodbatlas_custom_dns_configuration_cluster_aws" }}
+    {{ else if eq .Name "mongodbatlas_search_index" }}
+    {{ else if eq .Name "mongodbatlas_team" }}
+    {{ else if eq .Name "mongodbatlas_project_invitation" }}
+    {{ else if eq .Name "mongodbatlas_org_invitation" }}
+    {{ else if eq .Name "mongodbatlas_project_api_key" }}
+    {{ else if eq .Name "mongodbatlas_global_cluster_config" }}
+    {{ else if eq .Name "mongodbatlas_network_container" }}
+    {{ else if eq .Name "mongodbatlas_custom_db_role" }}
+    {{ else if eq .Name "mongodbatlas_federated_settings_identity_provider" }}
+    {{ else if eq .Name "mongodbatlas_cloud_backup_snapshot_restore_job" }}
+    {{ else if eq .Name "mongodbatlas_private_endpoint_regional_mode" }}
+    {{ else if eq .Name "mongodbatlas_auditing" }}
+    {{ else if eq .Name "mongodbatlas_encryption_at_rest" }}
+    {{ else if eq .Name "mongodbatlas_ldap_configuration" }}
+    {{ else if eq .Name "mongodbatlas_teams" }}
+    {{ else if eq .Name "mongodbatlas_serverless_instance" }}
+    {{ else if eq .Name "mongodbatlas_maintenance_window" }}
+    {{ else if eq .Name "mongodbatlas_ldap_verify" }}
+    {{ else if eq .Name "mongodbatlas_third_party_integration" }}
+    {{ else if eq .Name "mongodbatlas_x509_authentication_database_user" }}
+    {{ else if eq .Name "mongodbatlas_privatelink_endpoint_service_data_federation_online_archive" }}
+    {{ else }}
+        {{ tffile (printf "examples/%s/main.tf" .Name )}}
+    {{ end }}
+{{ end }}
+
+{{ if .SchemaMarkdown }}{{ .SchemaMarkdown | trimspace }}{{ end }}
+
+# Import 
+## Add the import
+
+For more information see: [MongoDB Atlas API - Search Node](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Atlas-Search/operation/createAtlasSearchDeployment) Documentation.
```

Comment:
I believe we can remove this from the default template right? Not sure if we are capable of leaving this generic. Applies to the data source template as well.

Reply:
> I believe we can remove this from the default template right? Not sure if we are capable of leaving this generic.
we can have the import autogenerated if we define an `import.txt` in the example folder. similar to https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1394/files#diff-ab6cba42e3e4a9f3642faec48e42d41f668902c8ffa3041a36bbc56e01c59913. What do you think? 

> applies to the data source template as well.

data sources do not have the import